### PR TITLE
{lang}[GCCcore/9.3.0] moonjit v2.2.0

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f97f876737af5eb64bfeb9c68c92cab7 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node124.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/12ed332f75cedc62e5fdd2aac050734f for a full test report.

**Review by lexming — CHANGES_REQUESTED** [review]
The build of `moonjit` works well. I just have a minor remark about the sanity checks, which can be improved. Please sea below

File: easybuild/easyconfigs/m/moonjit/moonjit-2.2.0-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+dependencies = [
+    ('ncurses', '6.2'),
+    ('libreadline', '8.0'),
+]
+
+skipsteps = ['configure']
+installopts = 'PREFIX=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ["bin/luajit"],
```
[the remark this answers is not in the document]
Comment (furstj) [reply easybuild/easyconfigs/m/moonjit/moonjit-2.2.0-GCCcore-9.3.0.eb]: You are right, I will include it in https://github.com/easybuilders/easybuild-easyconfigs/pull/11163/commits/857b822cf9135768fcd020d55d7e67a0334b0a04

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node312.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4bf9ad8b408efd0d4501b102a18234fd for a full test report.

**Comment by lexming** [comment]
Going in, thanks @furstj !

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node124.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/48e1df0425523c93837237e66740f7b6 for a full test report.

**Pull request merged**